### PR TITLE
chore(OWNERS): move michelleN to emeritus

### PR DESCRIPTION
**Pull request by michelleN**
Thank you.

### CODE REVIEW

**Review by bacongobbler — APPROVED** [review]
❤️

File: OWNERS [hunk 1]
```diff
@@ -6,13 +6,13 @@ maintainers:
...
   - marckhouzam
   - mattfarina
-  - michelleN
   - prydonius
   - SlickNik
   - technosophos
   - thomastaylor312
   - viglesiasce
 emeritus:
+  - michelleN
```
Comment (hickeyma) [inline OWNERS:15]: nit: alphabetical order 🤣

**Review by hickeyma — COMMENTED** [review]
Small nit @michelleN

File: OWNERS (hunk 1 above)
Comment (michelleN) [reply OWNERS]: fixed! thnx!

**Review by hickeyma — APPROVED** [review]
Thanks for fixing @michelleN . Sad to see you go.

**Review by thomastaylor312 — APPROVED** [review]
Bye @michelleN! We'll miss you 😭

**Comment by bridgetkromhout** [comment]
Thanks for all you've done for Helm, @michelleN! 💖

**Comment by hickeyma** [comment]
@michelleN You can merge away when you feel like it!

**Pull request merged**